Doc/library/os.rst

```diff
@@ -3444,6 +3444,14 @@ written in Python, such as a mail server's external command delivery program.
    setting of the effective UID and GID. This argument corresponds to the C
    library :c:data:`POSIX_SPAWN_RESETIDS` flag.
 
+   If the *setsid* argument is ``True``, it will create a new session ID
+   for `posix_spawn`.The calling process is the leader of the new session (i.e.,
+   its session ID is made the same as its process ID). The calling process also
+   becomes the process group leader of a new process group in the session (i.e.,
+   its process group ID is made the same as its process ID). *setsid* requires :c:data:
+   `POSIX_SPAWN_POSIX_SPAWN_SETSID`or :c:data:`POSIX_SPAWN_POSIX_SPAWN_SETSID_NP`
+   flag.Otherwise, :exc:`NotImplementedError` is raised.
```

Comment:
`setsid` parameter doesn't affect the calling process in any way, so it's incorrect to say that "The calling process is the leader of the new session", etc. Your current text looks like docs for `os.setsid` instead of `setsid` parameter of `os.posix_spawn`.